backend/src/mongodb/schemas/stockSchema.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
```

Comment:
What is the point of this? I don’t think schema is being used?

Reply:
i use it for the _id object where I store the user id. I specify that the object is like a mongodb objectId type.
I based this off of this:
https://mongoosejs.com/docs/populate.html

Reply:
I think you can use Schema({ stockName: {},....}) instead then. Not sure if that will work though?

Reply:
that works, new commit changes that